Posit Infrastructure/Overnight Sessions.md

```diff
@@ -6,13 +6,13 @@ This document aims to provide users with information on running Posit Workbench
 
 ## Background
 
-An automated process has been put in place to close sessions in Posit Workbench at 9pm every evening.
+An automated process has been put in place to close sessions in Posit Workbench every evening. During **Daylight Saving Time**, sessions will close at **8pm**.
```

Comment:
The non DST time was dropped.
```suggestion
An automated process has been put in place to close sessions in Posit Workbench every evening at 9pm. During **Daylight Saving Time**, sessions will close at **8pm**.
```

Reply:
I don't see much value in keeping the 9 pm reference in the meantime. Less is more.